### PR TITLE
Add user help links for "NPI/UMPI" form labels

### PR DESCRIPTION
**Pull request by PaulMorris**
These labels appear in the enrollment steps in two places: on the individual provider 'Practice Info' page, and the organization provider 'Individual Member Info' page.

Tested by checking that the "?" links appear correctly on these pages and, when clicked, open the "What is an NPI?" modal.

Issue #422 Review documentation to add [...] help links

### CODE REVIEW

**Comment by PaulMorris** [comment]
Revised to add two instances, one on the Personal Care Assistant > Agency Info page. The other in a hidden template in member_information.jsp.

**Comment by jasonaowen** [comment]
It looks like the individual's group practice is not included:

group-practice-no-npi

Do you expect that to be included @PaulMorris? Or are you planning on adding it to that page separately?

**Comment by jasonaowen** [comment]
The help link in an organization's individual NPI seems not to work:

individual-member-npi

It doesn't seem to be getting the event listener attached, presumably because it's created by clicking "Add Another Individual Member" and isn't available at page load time.

**Comment by PaulMorris** [comment]
Thanks @jasonaowen.  I've pushed a new revision to address the issues you identified.  It's ready for review.

> It looks like the individual's group practice is not included:

Good catch!  This is now included.  (It slipped through the cracks because I forgot to cover both 'yes' and 'no' for "Do you maintain your own private practice?")

> The help link in an organization's individual NPI seems not to work:

Another good catch!  Now fixed.  It was indeed that the event listener had to be added on click of "Add Another Individual Member".  (I missed this before because I was checking it by editing a draft enrollment application rather than entering a new one, and forgot to try adding an additional individual member to the one that already existed in it.)

> I'm not sure what you mean by "a hidden template in member_information.jsp" - is that unused code?

Ah, I just meant the hidden "template" part of that html page / jsp template that is cloned on click of "Add Another Individual Member" to add additional forms for new members to the page.

File: psm-app/cms-web/WebContent/WEB-INF/pages/provider/enrollment/steps/pageTemplates/default/member_information.jsp [hunk 1]
```diff
@@ -28,7 +28,11 @@
             <div class="row requireField">
                 <c:set var="formName" value="_16_npi_${status.index - 1}"></c:set>
                 <c:set var="formValue" value="${requestScope[formName]}"></c:set>
-                <label for="${formIdPrefix}_${formName}">NPI/UMPI<span class="required">*</span></label>
+                <label for="${formIdPrefix}_${formName}">
+                  NPI/UMPI
+                  <span class="required">*</span>
+                  <a href="javascript:" class="userHelpLink NPIdefinition">?</a>
```
[the remark this answers is not in the document]
Comment (PaulMorris) [reply psm-app/cms-web/WebContent/WEB-INF/pages/provider/enrollment/steps/pageTemplates/default/member_information.jsp]: This was just a typo on my part leaving out the semicolon on `javascript:;`.  That is an interesting debate.  `javascript:;` is a pretty common convention that I've seen and used for this, but it looks like there may be better options.  (I stayed with `javascript:;` in the latest revision.)

**Comment by PaulMorris** [comment]
This revision avoids duplicate click event handlers from being attached to user help links when additional user help links are added dynamically to a page.  It also includes refactoring of the common.js file to put the top level functions at the top level, which is a more idiomatic way of organizing JS code.

File: psm-app/frontend/src/main/js/common.js [hunk 2]
```diff
@@ -1,135 +1,142 @@
...
+/**
+* Add a click handler function to a user help modal link.
+* Title is optional; is absent (undefined) to show a single help item.
+*
+* @param helpLinkSelector {string} - Selector for the help link.
+* @param helpDocsPath {string} - Path to the help doc html file.
+* @param helpItemIds {array of strings} - IDs of the source help items.
+* @param title {string or undefined} - Title to use with multiple help items.
+*/
+function addUserHelpClickHandler(
```
Comment (jasonaowen) [inline psm-app/frontend/src/main/js/common.js]: Do you want to rename this to `setUserHelpClickHandler` or similar? If so, I think this PR is the time.

Comment (PaulMorris) [reply psm-app/frontend/src/main/js/common.js]: Good idea, will do.

**Comment by PaulMorris** [comment]
@jasonaowen This revision adds a commit "Rename addUserHelpClickHandler to setUserHelpClickHandler".

**Pull request merged**